apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/doedshendelse/DoedshendelseDao.kt

```diff
@@ -26,19 +35,27 @@ class DoedshendelseDao(val connection: () -> Connection) {
                 setTidspunkt(7, doedshendelse.endret)
                 setString(8, doedshendelse.status.name)
             }.executeUpdate()
+            if (shouldCloseConnection()) {
```

Comment:
Har du vurdert om du må fange exception fra executeUpdate()?

Reply:
For å ikke legge connections? Ja burde nok gjort det.
Men gikk for en annen variant her https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/3819/commits/6bff52cabdbd37c5455e716abf53587ae6f2bd3d
Den wrapper det bare